index.html

```diff
@@ -284,45 +281,75 @@ <h2>Skills</h2>
     </section>
 
     <section id="contact-section" class="flex-container">
-      <h1>
+      <p>
         I'm always interested in hearing about new projects, so if you'd like to
         chat please get in touch.
-      </h1>
+      </p>
 
       <form action="https://formspree.io/f/xeqdoera" method="POST">
-        <input type="text" name="fullname" id="fullname" placeholder="Full name" maxlength="30" required>
-        <input type="email" name="email" id="email" placeholder="Email address" required>
-        <textarea name="message" id="massage" maxlength="500" placeholder="Enter text here" required ></textarea>
+        <label for="fullname">Full Name</label>
+        <input
+          type="text"
+          name="fullname"
+          id="fullname"
+          placeholder="Full name"
+          maxlength="30"
+          required
+        />
+        <label for="email">Email</label>
+        <input
+          type="email"
+          name="email"
+          id="email"
+          placeholder="Email address"
+          required
+        />
+        <label for="massage">Message</label>
+        <textarea
+          name="message"
+          id="massage"
+          maxlength="500"
+          placeholder="Enter text here"
+          required
+        ></textarea>
         <button class="call-to-action" type="submit">Get in touch</button>
       </form>
```

Comment:
- [ ] Great Job so far, for a better UX  any fields that are required/mandatory should be indicated to the user, kindly add something that tells if the input is required or not, you can add `(required)` or `*` in your placeholders or labels

Reply:
### Hi @ichala 👋

Thank you so much for the feedback, we will do so.

Reply:
Hey @ichala  👋 thank you for the feedback. I have a question. 
So, the `required` attribute has already been added to all three inputs, is it better to add both the attribute and the `*`?

Reply:
@laiifuu 
the user will not know it's required until he clicks on submit , so just a simple message in the label or place holder show him that the input is a required field example : 

![image](https://user-images.githubusercontent.com/89282221/187539611-ca6c94e4-81e1-45d4-bf34-509661327ffa.png)


let me know if you need more explanation , keep rocking 👏

Reply:
@ichala Thank you 😃